main/cmds.go

```diff
@@ -92,22 +94,37 @@ func migrateToMostRecentSequence(ctx *log.Context, h HandlerEnvironment, seqNum
 	// do not have invent another method.  The CustomScript extension should have been using this
 	// from the beginning, but it was not.
 	//
-	computedSeqNum, err := FindSeqNumStatus(h.HandlerEnvironment.StatusFolder)
+	ctx.Log("event", "migrate to mrseq", "message", fmt.Sprintf("cannot find mrseq at %v", mostRecentSequence))
+	fout, err := os.Create(mostRecentSequence)
+	defer fout.Close()
 	if err != nil {
-		// If there was an error, the sequence number is zero.
 		ctx.Log("event", "migrate to mrseq", "error", err)
-		return
 	}
 
-	fout, err := os.Create(mostRecentSequence)
-	if err != nil {
-		ctx.Log("event", "migrate to mrseq", "error", err)
-		return
+	if seqNum != 0 {
+		computedSeqNum := seqNum
+		_, err := fout.WriteString(fmt.Sprintf("%v", computedSeqNum))
+		if err == nil {
+			ctx.Log("event", "migrated to mrseq", "message", fmt.Sprintf("migrated mrseq to %v from config seqnum", computedSeqNum))
```

Comment:
We're not really migrating, right? Mrseq was deleted, so we're doing our best to recreate it.

Reply:
Yup... was just keeping current log format. Maybe, "recreating mrseq" would be better?